backend/src/paymaster/index.ts

```
@@ -738,6 +776,101 @@ export class Paymaster {
+        preVerificationGas: BigNumber.from(packedUserOp.preVerificationGas).toHexString(),
+        verificationGasLimit: BigNumber.from(userOp.verificationGasLimit).toHexString(),
+        callGasLimit: BigNumber.from(userOp.callGasLimit).toHexString(),
+        paymasterVerificationGasLimit: userOp.paymasterVerificationGasLimit,
+        paymasterPostOpGasLimit: userOp.paymasterPostOpGasLimit,
+      }
+      return returnValue;
+    } catch (err: any) {
+      if (err.message.includes("Quota exceeded"))
+        throw new Error('Failed to process request to bundler since request Quota exceeded for the current apiKey')
+      if (log) log.error(err, 'signMultiTokenPaymasterV07');
+      throw new Error('Failed to process request to bundler. Please contact support team RawErrorMsg:' + err.message)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)